Description: Fix argparse usage for Python 2.7.9
 The behaviour of set_defaults appears to have
 regressed in python 2.7.9; this patch works around
 this problem by providing the default for 'cluster'
 alongside the --cluster option.
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/ceph/+bug/1413321
Forwarded: no

--- a/src/ceph-disk
+++ b/src/ceph-disk
@@ -3618,6 +3618,7 @@ def parse_args(argv):
     parser.set_defaults(
         # we want to hold on to this, for later
         prog=parser.prog,
+        cluster='ceph',
         )
 
     subparsers = parser.add_subparsers(
